components/desktop/gnome3/seahorse/patches/02-gnupg.patch

```diff
@@ -0,0 +1,11 @@
+--- seahorse-3.20.0/configure.ac.orig	2019-12-19 03:12:08.257265619 +0000
++++ seahorse-3.20.0/configure.ac	2019-12-19 03:12:32.896088202 +0000
+@@ -9,7 +9,7 @@
+ GTK_REQ=3.4.0
+ GTK_MAX=GTK_VERSION_3_4
+ 
+-GNUPG_ACCEPTED="2.0.12 2.1.4"
++GNUPG_ACCEPTED="2.0.12 2.1.4 2.2.17"
```

Comment:
Here's one more issue here - it seems gpgme should be updated to cope with new gpg statuses - https://github.com/GNOME/seahorse/commit/cdfc5b297d7420e47b9c973e8b8cb1b0fb576421